packages/core/src/Cardano/types/Governance.ts

```diff
@@ -110,17 +110,10 @@ export type VotingProcedure = {
   anchor: Anchor | null;
 };
 
-export type VotePayload = {
-  governanceActionId: GovernanceAction;
-  votingProcedure: VotingProcedure;
-};
-
-export type Voter = Credential;
+// Do we need to know what type of credential the Voter is?
```

Comment:
I vote "yes". Depending on the type of credential, we determine if we can sign it and what is the derivation path.
```
; Constitutional Committee Hot KeyHash: 0
; Constitutional Committee Hot ScriptHash: 1
; DRep KeyHash: 2
; DRep ScriptHash: 3
; StakingPool KeyHash: 4
voter =
  [ 0, addr_keyhash
  // 1, scripthash
  // 2, addr_keyhash
  // 3, scripthash
  // 4, addr_keyhash
  ]
```